pynsee/localdata/_get_insee_local_onegeo.py

```diff
@@ -19,14 +18,16 @@ def _get_insee_local_onegeo(variables, dataset_version, nivgeo, codegeo):
         variables, dataset_version, nivgeo, codegeo, modalite
     )
 
-    request = _request_insee(api_url=link, file_format="application/json;charset=utf-8")
+    request = _request_insee(
+        api_url=link, file_format="application/json;charset=utf-8"
+    )
 
     data_request = request.json()
 
     Cellule = data_request["Cellule"]
     Variable = data_request["Variable"]
     Croisement = data_request["Croisement"]
-    Zone = data_request["Zone"]
+    # Zone = data_request["Zone"]

```

Comment:
remove?

Reply:
Sure. (I might also have done it in the "prune" PR 🙄 )